tests/beacon/test_helpers.py

```diff
@@ -34,10 +39,20 @@
     get_new_validator_registry_delta_chain_tip,
     _get_shard_committees_at_slot,
     get_block_committees_info,
+    get_pubkey_for_indices,
+    generate_aggregate_pubkeys,
+    verify_vote_count,
+    verify_slashable_vote_data_signature,
+    verify_slashable_vote_data,
     is_double_vote,
     is_surround_vote,
 )
+import eth._utils.bls as bls
 
+from hypothesis import (
+    given,
+    strategies as st,
+)
```

Comment:
nitpick: move `hypothesis` importing to about L6 (between `import random` and `from eth_utils import (...)` because it's a third-party package.  (https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/style-guide.md#overall-ordering)

Reply:
gotcha good call